the-lost-penguin/Assets/Scripts/PlayerStats.cs

```diff
@@ -62,10 +63,10 @@ void Update()
         //both enemy and item scripts should have an exp field to be used
 
         //testing, remove when done
-        // if(Input.GetKeyDown(KeyCode.J))
-        // {
-        //     AddExp(1);
-        // }
+        if(Input.GetKey(KeyCode.J))
+        {
+            AddExp(1);
```

Comment:
What is this for? Should the player be able to spam "J" and just get more exp?

Reply:
debugging purposes. I forgot to comment it out